selenium_tests/cross_listing/page_objects/cross_listing_main_page.py

```diff
@@ -69,9 +69,11 @@ def delete_cross_listing_pairing(self, data_xlist_map_id):
 
     def is_locator_text_present(self, expected_text):
         """
-        Check to that locator text appears on the confirmation page
+        Check that the locator text appears in the confirmation text box
         """
         try:
+            WebDriverWait(self._driver, 60).until(lambda s: s.find_element(
```

Comment:
Good point.  I can lower to 30s, but let's try with 60s first, because I want to make sure that this actually fixes the failure -- in case Jenkins was just slow and couldn't find the element without the explicit waits (since this hadn't been an issue when I ran the file individually).  Can totally revisit the length of waits.  Thanks!